Header header

bool found_tennis

geometry_msgs/PointStamped ball_pose

std_msgs/Float64 distance